components/actionbar/stories/actionbar.stories.js

```diff
@@ -80,12 +80,10 @@ export default {
 
 export const Default = ActionBarGroup.bind({});
 Default.args = {
-	customPopoverStyles: {
-		// Prevent the popover from being rendered offscreen
-		"--spectrum-popover-animation-distance": "-10px",
-		"inline-size": "600px",
-		"max-inline-size": "100%",
-	}
+	// customPopoverStyles: {
```

Comment:
Should we delete these commented out blocks now or were we keeping them around for when we revisit popover?

Reply:
Great minds!  I had just deleted this after your review.